srcpkgs/python-Twisted/template

```diff
@@ -31,25 +31,3 @@ alternatives="
 post_install() {
```

Comment:
You need to add the `mv` here as well, no? To add the `2` suffix?

Reply:
When I tested, it didn't need the mv for it to install, but it may need it for the alternatives to work correctly, so I added it. I confess to not fully understand how alternatives works...